### ISSUE TITLE
Get ldapauthenticator released and bumped here

### ISSUE THREAD

**Issue by consideRatio** [jupyterhub/zero-to-jupyterhub-k8s#1561]
https://github.com/jupyterhub/ldapauthenticator latest release is very outdated and I hoped to get it bumped for 0.9.0 as it apparently contains quite a bit of fixes. It requires additional work from maintainers though, and we are very low on maintainers on that repo.

**Comment by manics** [comment]
What stopping us from releasing the current master now?

**Comment by consideRatio** [comment]
I don't know fully, i lack understanding about ldap in general, so I cannot even answer that ;)

**Comment by manics** [comment]
There are several open issues and PRs, but I'm not aware of any regressions in `master` that prevent a release.

How about we release the current master? Now that the automated deployment work is in place it should be easy to push out smaller releases after.

@yuvipanda @dhirschfeld

**Comment by consideRatio** [comment]
I spent some time to do what I could:
1. Reviewed / Merged some PRs
2. Fixed merge conflicts etc and prepared https://github.com/jupyterhub/ldapauthenticator/pull/106 to be merged - review help wanted!
3. Wrote a changelog - review help wanted for https://github.com/jupyterhub/ldapauthenticator/pull/158!

@manics has already done work to update our CD to PyPI when pushing tags and made a RELEASE.md file, and I've made sure we have a deployment token available in the PYPI_PASSWORD environment variable. So, after review/merge on stuff above, I think we should make the 1.3.0 release of ldapauthenticator.
